client/state/partner-portal/stored-cards/actions.ts

```diff
@@ -41,7 +41,9 @@ export const fetchStoredCards = ( paging: { startingAfter: string; endingBefore:
 		} );
 };
 
-export const deleteStoredCard = ( card: PaymentMethod ) => ( dispatch: Dispatch< AnyAction > ) => {
+export const deleteStoredCard = ( card: PaymentMethod, primary_payment_method_id?: string ) => (
```

Comment:
We can use camelCase for this parameter.

Reply:
Thanks! Fixed in e121f57fab0a56c5653bf4d67a99a44958db2a81